test/Ocelot.AcceptanceTests/RoutingTests.cs

```diff
@@ -589,7 +589,8 @@ public void should_return_not_found()
                 .And(x => _steps.GivenThereIsAConfiguration(configuration))
                 .And(x => _steps.GivenOcelotIsRunning())
                 .When(x => _steps.WhenIGetUrlOnTheApiGateway("/products/"))
-                .Then(x => _steps.ThenTheStatusCodeShouldBe(HttpStatusCode.NotFound))
+                .Then(x => _steps.ThenTheStatusCodeShouldBe(HttpStatusCode.OK))
+                .Then(x => ThenTheDownstreamUrlPathShouldBe("/products"))

```

Comment:
@raman-m the change in this test reflects the correct desired behavior of matching empty places holders if they appear  on upstream path and there is a route that matches

Reply:
You need to convert this `Fact` to `Theory`